default.xml

```diff
@@ -14,37 +14,51 @@
 
     <!-- We assume this manifest is in the same path as all the libraries. If you are
      copying this manifest as a template then you probably want the fetch path
-     for this to be '../seL4' instead of '.' -->
-    <remote name="seL4"
-        fetch="."
-    />
+     for this to be '../seL4' instead of '.' added to .repo/local_manifest/ -->
+    <remote name="current" fetch="."/>
 
     <!-- default revision for each project, and name of the remote -->
-    <default revision="master"
-        remote="seL4"
-    />
-
-    <!-- list of 'projects' (i.e repositories), path is relative to the path in remote above -->
-    <project name="seL4.git" path="kernel" revision="experimental" />
-    <project name="common-tool.git" path="tools/common"/>
-    <project name="kbuild-tool.git" path="tools/kbuild"/>
-    <project name="elfloader-tool.git" path="tools/elfloader"/>
-    <project name="libmuslc.git" path="libs/libmuslc"/>
-    <project name="libsel4muslcsys.git" path="libs/libsel4muslcsys"/>
-    <project name="libsel4platsupport.git" path="libs/libsel4platsupport"/>
-    <project name="elfloader-tool.git" path="libs/elfloader"/>
-    <project name="libsel4allocman.git" path="libs/libsel4allocman"/>
-    <project name="libsel4vka.git" path="libs/libsel4vka"/>
-    <project name="libsel4vspace.git" path="libs/libsel4vspace"/>
-    <project name="libsel4utils.git" path="libs/libsel4utils"/>
-    <project name="libcpio.git" path="libs/libcpio"/>
-    <project name="libelf.git" path="libs/libelf"/>
-    <project name="libsel4simple.git" path="libs/libsel4simple"/>
-    <project name="libsel4simple-stable.git" path="libs/libsel4simple-stable"/>
-    <project name="libplatsupport.git" path="libs/libplatsupport"/>
-    <project name="libutils.git" path="libs/libutils"/>
-
-    <project name="refos.git" path="projects/refos">
+    <default revision="master" remote="current"/>
```

Comment:
Mostly artifacts of my being completely new to repo/and before I uploaded having some repos on current/some referencing github